src/Features/Auth/Login/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+import FormComponent from '../../../../components/FormComponent';
+
+function LoginForm() {
+  const [loginForm, setLoginForm] = useState({});
+
+  const loginClickHandler = () => {};
+
+  const textFiledChangeHandler = (e) => {
+    const { value, name } = e.target;
+
+    setLoginForm((prev) => {
+      return {
+        ...prev,
+        [name]: value,
+      };
+    });
+  };
+  const loginInputs = {
+    fields: [
+      {
+        required: true,
+        label: 'UserName',
+        name: 'username',
+        type: 'text',
+        value: loginForm?.username || '',
+        changeHandler: textFiledChangeHandler,
+      },
+      {
+        required: true,
+        label: 'Password',
+        name: 'password',
+        type: 'password',
+        minlength: '6',
+        value: loginForm?.password || '',
+        changeHandler: textFiledChangeHandler,
+      },
+    ],
+  };
+  const loginButtons = {
+    button: [
+      {
+        type: 'submit',
+        name: 'login',
+        minWidth: '100%',
+        clickHandler: loginClickHandler,
+      },
+    ],
+  };
+
+  return (
+    <div>
+      <FormComponent
+        basicButtons={loginButtons}
+        forgotPassword="Forgot Password?"
```

Comment:
I have previously asked you not to used specific titles in common components try to use generic name like link or secondary links .. because forget password is a link title ..